### DISCUSSION TITLE
how to disable some parts like physics and 3d

### DISCUSSION THREAD

**Discussion by meehjeevan** [axmolengine/axmol#1922] [Q&A]
How to disable some parts like physics and 3d features which is not been used in my game so that I can reduce build time and package size.

**Comment by rh101** [comment]
https://github.com/axmolengine/axmol/wiki/Extensions

Take a look in these files for things you can disable: 
https://github.com/axmolengine/axmol/blob/dev/extensions/CMakeLists.txt
https://github.com/axmolengine/axmol/blob/dev/core/CMakeLists.txt

Also search for `AX_ENABLE_` in `CMakeLists.txt` files within the engine source tree, and you'll be able to see all the optional code you can enable/disable.
